superset-frontend/src/explore/components/controls/DndColumnSelectControl/ColumnSelectPopover.tsx

```diff
@@ -417,6 +482,13 @@ const ColumnSelectPopover = ({
         >
           {t('Save')}
         </Button>
+        <i
```

Comment:
Could we move the whole resizing logic to a hook that would only return a react component `<i ... />`?